pubsub/pubsub.go

```diff
@@ -22,16 +22,30 @@ import (
 
 // PubSub is the interface for message buses.
 type PubSub interface {
+	BulkMessager
```

Comment:
As per my comment on https://github.com/dapr/components-contrib/pull/2024#discussion_r968381941 I wonder if this is best implemented as an optional interface. This way we don't force components to inherit the stub object and the runtime can do the check at ... runtime.

Reply:
Any decision regarding the struct will be handled as part of #2024 . Want to limit this PR specific to Kafka alone.

Reply:
This is done ...